.github/workflows/auto-updates.yml

```
@@ -88,6 +88,7 @@ jobs:
       - name: Generate dictionary (StarDict)
         run: |
           pyglossary --ui=none data/${{ matrix.locale }}/dict-${{ matrix.locale }}.df dict-data.ifo
+          sed -i 's/bookname=.*/dict-${{ matrix.locale }}-${{ matrix.locale }}/' dict-data.ifo
```

Shouldn't we keep the "bookname" part?

```suggestion
          sed -i 's/bookname=.*/bookname=dict-${{ matrix.locale }}-${{ matrix.locale }}/' dict-data.ifo
```

---

opps, fixed.